lib/src/tools/analyze_tool.dart

```diff
@@ -74,7 +74,7 @@ class AnalyzeTool extends DevTool {
 
   @override
   FutureOr<int> run([DevToolExecutionContext context]) {
-    useDartAnalyze ??= false;
+    useDartAnalyze ??= true;
```

Comment:
Maybe we should update the doc comment on `useDartAnalyze` to indicate that we will actually use `dart analyze` by default.

Reply:
Oh that's a good idea!